### PR TITLE
[DCOS-56662] Cassandra bump SDK to v 0.57.0-rc-2

### PR DESCRIPTION
**Pull request by rathohit311356**
## What changes were proposed in this pull request?
Resolves: [[DCOS-56662] [Cassandra] Bump SDK to version 0.57.0](https://jira.mesosphere.com/browse/DCOS-56662)
Changes include below features part of 0.57.0-rc2 sdk update:

- Task Backoff Delay
- Shared Hierarchial Memory Support 
- Quota Support
For more details please refer: https://github.com/mesosphere/dcos-commons/releases/tag/0.57.0-rc2

## How were these changes tested?
All the changes were implemented one by one and tested whether respective changes reflecting or not locally , service installed and tested its normal behaviour

Below are the screen shot for quota support and backoff delay
quotasupport
back-off delay

### CODE REVIEW

File: sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawTask.java [hunk 1]
```diff
@@ -99,6 +107,10 @@ public Integer getMemory() {
     return memory;
   }
 
+  public Integer getShmsize() {
+    return shmsize;
```
Comment (samvantran) [inline sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawTask.java]: Hey @kaiwalyajoshi and @kvish looks like some additional changes were needed to get things working on sdk-0570-rc1

cc @rathohit311356

Comment (kaiwalyajoshi) [reply sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawTask.java]: This shouldn't be required with the [rc2 release](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.0-rc2).

**Comment by kaiwalyajoshi** [comment]
I think we can close this as rc2 release should supercede this.

**Comment by samvantran** [comment]
@kaiwalyajoshi sorry title didn't reflect reality. I actually merged in the sdk-rc branch (with rc2) in commit b77bead. So we're mostly in a holding pattern until you guys release a GA version and we'll update this PR accordingly.

**Comment by rathohit311356** [comment]
Closing this as its not required now

**Pull request closed without merging**